src/averageAges.js

```diff
@@ -54,11 +92,38 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // 1. find a mother of each person (or only for men)
-  // 2. keep people who have mothers in the array
-  // 3. calculate the difference child.born - mother.born
-  // 4. return the average value
+  let childrenWithMothers = people
+    .filter(child => child['mother'] !== null);
```

Comment:
But better will be to check if we have mother with such name in our `people` array